test/integration/domain/oracle/index.test.js

```diff
@@ -39,39 +41,285 @@ describe('Oracle', () => {
     await Db.disconnect()
   })
 
-  it('creates an oracle', async () => {
-    // Arrange
-    const payload = {
-      isDefault: true,
-      currency: 'AUD',
-      oracleIdType: 'MSISDN',
-      endpoint: {
-        value: 'http://localhost:8444',
-        endpointType: 'URL'
-      }
+  const createHeaders = {
+    accept: 'application/vnd.interoperability.participants+json;version=1',
+    'cache-control': 'no-cache',
+    date: '',
+    'content-type': 'application/vnd.interoperability.participants+json;version=1.0',
+    host: '127.0.0.1:4003',
+    'accept-encoding': 'gzip, deflate',
+    'content-length': 164,
+    connection: 'keep-alive'
+  }
+  const testSpan = EventSdk.Tracer.createSpan('createOracle service')
+
+  const ORACLE_AUD_MSISDN_URL_PAYLOAD = {
```

Comment:
Please use `camelCase` for objects like this.

Reply:
done